edge/pkg/metamanager/process.go

```diff
@@ -477,3 +478,46 @@ func (m *metaManager) runMetaManager() {
 		}
 	}()
 }
+
+func sendToEdgedByNodeNameOrHostIP(message model.Message, resType string, isSync bool) {
+	switch resType {
+	case "pod":
+		if podMatchByNodeNameOrHostIP(message, metaManagerConfig.Config.NodeName) {
+			sendToEdged(&message, isSync)
+		}
+	default:
+		sendToEdged(&message, isSync)
+	}
+}
+
+func podMatchByNodeNameOrHostIP(message model.Message, nodeName string) (isMatched bool) {
+	content := message.GetContent()
```

Comment:
We can json.Unmarshal  the content to pod directly.

Reply:
OK, modified